openshift-servicemesh/instance/base/servicemesh-controlplane.yaml

```diff
@@ -1,31 +1,57 @@
 apiVersion: maistra.io/v2
 kind: ServiceMeshControlPlane
 metadata:
-  name: prod-tenant-01
+  name: prod-tenant-02
```

Comment:
We are creating a "default" base template here in this file, so I think it makes sense to keep names as basic as possible. So maybe this control plane can be called `default` or `basic`?